### ISSUE TITLE
Unable to change length of generated password with TalkBack

### ISSUE THREAD

**Issue by SaschaCowley** [bitwarden/mobile#801]
## Describe the Bug

Using TalkBack on Android, it isn't possible to change the length of the generated password.

## Steps To Reproduce

1. Activate Talkback (Settings -> Accessibility)
2. In the Bitwarden app, select the 'Generator' tab.
3. Try to adjust the 'length' slider, either with the volume keys, or via the 'Edit seek control level' option in the 'Seek control' menu of TalkBack's 'Local context menu'.

## Expected Result

The length value should change as the value of the slider changes, and as a result new passwords should be generated.

## Actual Result

The slider thumb moves, and TalkBack reports a new level, but the length value text does not change, nor does the generated password.

## Environment

- Device: Pixel 4
- Operating system: Android 10
- Build Version: 2.3.1 (2257)
- Is this a Beta release? Yes

## Additional Context

From a usability perspective, the slider should probably be replaced with a number input.
